kolibri/deployment/default/settings/base.py

```diff
@@ -195,5 +195,10 @@
 
 AUTH_USER_MODEL = 'kolibriauth.DeviceOwner'
 
+AUTH_ANONYMOUS_USER_MODEL = "kolibri.auth.models.KolibriAnonymousUser"
```

Comment:
`REST_FRAMEWORK["UNAUTHENTICATED_USER"]` is used by DRF, and `AUTH_ANONYMOUS_USER_MODEL` is used by `kolibri.auth.middleware.CustomAuthenticationMiddleware`. For the latter, I can either hardcode the class into the middleware or have it as an app-level configuration setting. Since I can't imagine it being overridden, I'll just hardcode the import into the middleware for now.